index.html

```diff
@@ -92,8 +76,10 @@ <h1 class="name">Akumu Bavon</h1>
       <div class="image_dots"></div>
       <div class="bottom_geometry"></div>
     </section>
+    <div class="project_details" id="view_project">
 
-    <section class="project_section" id="Portfolio">
+    </div>
+    <section class="project_section can_dissapear" id="Portfolio">
```

Comment:
- [ ] Kindly please implement the modals on the projects section on the desktop view, as stated on the requirements:

![image](https://user-images.githubusercontent.com/57421823/151077796-eeac4e18-6b91-4e5e-a9f1-3fc8ef3d3454.png)

That way, your portfolio will have popups both on desktop and mobile.  You could reuse the logic that you already implemented for the mobile! 💪